bundles/org.openhab.persistence.jdbc/README.md

```diff
@@ -62,7 +62,7 @@ This service can be configured in the file `services/jdbc.cfg`.
 | numberDecimalcount          | 3                                                            |    No     | for Itemtype "Number" default decimal digit count            |
 | tableNamePrefix             | `item`                                                       |    No     | table name prefix. For Migration from MySQL Persistence, set to `Item`. |
 | tableUseRealItemNames       | `false`                                                      |    No     | table name prefix generation.  When set to `true`, real item names are used for table names and `tableNamePrefix` is ignored.  When set to `false`, the `tableNamePrefix` is used to generate table names with sequential numbers. |
-| tableCaseSensitiveItemNames | `false`                                                      |    No     | table name case when `tableUseRealItemNames` is `true`. When set to `true`, item name case is preserved in table names and no suffix is used. When set to `false`, table names are lower cased and a numeric suffix is added. Please read [this](#case-sensitive-item-names) before enabling. |
+| tableCaseSensitiveItemNames | `false`                                                      |    No     | table name case when `tableUseRealItemNames` is `true`. When set to `true`, item name case is preserved in table names and no prefix or suffix is added. Please make sure to have a dedicated schema when using this option, since otherwise table name collisions are more likely to happen. When set to `false`, table names are lower cased and a numeric suffix is added. Please read [this](#case-sensitive-item-names) before enabling. |
```

Comment:
"table name case when `tableUseRealItemNames` is **`true`."**

I find this hard to understand, isn't it better to rephrase it to:
"This setting is only applicable when `tableUseRealItemNames` is **`true`."*"

Or does it mean soimething different?

Reply:
Correct, thanks - rephrased.